packages/zk-transfer-web/public/index.html

```diff
@@ -22,13 +22,22 @@
     <title>Zero Knowledge</title>
     <script type="text/javascript">
       // window["RINKEBY.ETH_URL"] = "https://ganache.arcology.nyc:8547"
-      window["RINKEBY.DB_URL"] = "https://ganache.arcology.nyc:7001"
-      window.INFURA_PROJECT_ID = "590812d0023e4798aba2a764c477f504"
-      window.NODE_ENV = 'RINKEBY'
+      window["RINKEBY.DB_URL"] = "https://ganache.arcology.nyc:7001";
```

Comment:
to be consistent in style, let's use the `[]` notation consistently to assign to members of `window`, instead of mixing it with `.` notation.
```
window['RINKEBY.DB_URL'] = ...
window['INFURA_PROJECT_ID'] = ...
...
```